examples/src/main/java/com/example/ftclibexamples/SharedOdometry/AutoStaticRobotPose.java

```diff
@@ -17,17 +15,24 @@
 
     public static final double TRACKWIDTH = 14.31;
     public static final double CENTER_WHEEL_OFFSET = 0.477;
+    public static final double WHEEL_DIAMETER = 2.0;
+    public static final double TICKS_PER_REV = 8192;
+    public static final double DISTANCE_PER_PULSE = Math.PI * WHEEL_DIAMETER / TICKS_PER_REV;
```

Comment:
Gearing term needed

Reply:
Same as above, should be added as a comment or on TICKS_PER_REV, add a comment explaining the process on how to get it.